src/cephadm/cephadmlib/daemons/ceph.py

```diff
@@ -407,6 +410,55 @@ def create_daemon_dirs(self, data_dir: str, uid: int, gid: int) -> None:
         populate_files(ssl_dir, cert_files, uid, gid)
 
 
+def retrieve_version(container_id: str, container_command: str) -> str:
```

Comment:
Since this function is only used within this file naming it `_retrieve_version` would be cleaner as it says 'not part of the module's public api'.